File: app/src/main/java/com/faendir/lightning_launcher/multitool/fastadapter/DeletableModel.java
package com.faendir.lightning_launcher.multitool.fastadapter;

import android.content.Context;
import androidx.annotation.NonNull;

/**
 * @author F43nd1r
 * @since 13.10.2017
 */

public interface DeletableModel extends Model {
    String getUndoText(@NonNull Context context);

    void setName(String name);
}

File: app/src/main/java/com/faendir/lightning_launcher/multitool/fastadapter/DeletableModel.kt
package com.faendir.lightning_launcher.multitool.fastadapter

import android.content.Context

/**
 * @author F43nd1r
 * @since 13.10.2017
 */

interface DeletableModel : Model {
    fun getUndoText(context: Context): String
    override var name : String
}
